### PR TITLE
Add to calendar widget

### PR DESCRIPTION
**Pull request by christian-withjoy**
am i doing it right?

### CODE REVIEW

File: src/components/addToCalendarWidget/addToCalendarWidget.tsx [hunk 1]
```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
+import { css } from 'glamor';
+import { error } from 'util';
+import AddToCalendar from 'react-add-to-calendar';
+AddToCalendar.prototype.toggleCalendarDropdown = function (e) { return; };
```
Comment (mcorrigan89) [inline src/components/addToCalendarWidget/addToCalendarWidget.tsx]: What's going on here?

Comment (christian-withjoy) [reply src/components/addToCalendarWidget/addToCalendarWidget.tsx]: Calendar plugin that I'm wrapping includes a button and an automatic visibility toggle. I've hobbled it so it doesn't toggle.

Comment (mcorrigan89) [reply src/components/addToCalendarWidget/addToCalendarWidget.tsx]: Sweet that makes sense. Can you add that as a comment above the code too for future reference?

**Comment by christian-withjoy** [comment]
@mcorrigan89 OK updated pr to have storybook part

File: src/components/addToCalendarWidget/addToCalendarWidget.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+  var retval = {
+    [key]: {
+      //'content': `url(${icon})`,
+      ':after': {
+        backgroundImage: `url(${icon})`,
+        backgroundRepeat: 'no-repeat',
+      },
+    },
+  };
+  console.log(retval);
```
Comment (mcorrigan89) [inline src/components/addToCalendarWidget/addToCalendarWidget.tsx]: Looks good, just remove the console.logs and it's good to merge

File: src/components/addToCalendarWidget/addToCalendarWidget.tsx [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+      title: this.props.name,
+      description: this.props.details + (this.props.dressCode ? ' Dress Code: ' + this.props.dressCode : ''),
+      location: (this.props.location ? this.props.location : this.props.coupleNameString + "'s Wedding"),
+      startTime: this.props.dtstart,
+      endTime: endTime
+    }
+  }
+
+  render() {
+    console.log("rendering with props", this.props);
```
Comment (mcorrigan89) [inline src/components/addToCalendarWidget/addToCalendarWidget.tsx]: here too

**Pull request merged**